app/modules/ads/tracking/slot-tracker.js

```diff
@@ -36,3 +36,18 @@ export const registerViewabilityTracker = () => {
       trackingMethod: 'internal',
     })));
 };
+
+export const registerClickPositionTracker = () => {
+  const {
+    clickPositionTracker,
+    clickPositionTrackingMiddleware,
+  } = window.Wikia.adEngine;
+  const slotName = 'floor_adhesion';
+
+  clickPositionTracker
+    .add(clickPositionTrackingMiddleware)
+    .register(({ data }) => track(Object.assign(data, {
```

Comment:
Why object assign? Favour spread operator.

Reply:
Mobile-Wiki does not support spread operator.